variantgrid/settings/components/default_settings.py

```diff
@@ -957,6 +959,10 @@
 VARIANT_VCF_DB_PREFIX = "vg"
 VARIANT_MANUAL_CREATE = True
 VARIANT_MANUAL_CREATE_BY_NON_ADMIN = True
+# Below this size, variants are stored with ref/alt sequences. Above this threshold, they become
+# structural variants and use symbolic
+VARIANT_SYMBOLIC_ALT_SIZE = 1000
+VARIANT_SYMBOLIC_ALT_VALID_TYPES = {"<DEL>", "<DUP>", "<INS>", "<CNV>"}
```

Comment:
Not the biggest deal, but should all these be constants as they're referred a few times throughout.
They're small and standard enough that I'm not overly concerned if you'd prefer to leave them as strings.

Reply:
ok, did this just running tests